lib/internal/modules/esm/hooks.js

```diff
@@ -593,12 +602,12 @@ class HooksProxy {
     return body;
   }
 
-  makeSyncRequest(method, ...args) {
+  makeSyncRequest(transferList, method, ...args) {
     this.waitForWorker();
 
     // Pass work to the worker.
-    debug('post sync message to worker', { method, args });
-    this.#worker.postMessage({ method, args });
+    debug('post sync message to worker', { method, args, transferList });
+    this.#worker.postMessage({ method, args }, transferList);
```

Comment:
```suggestion
    this.#worker.postMessage({ __proto__: null, method, args }, transferList);
```
(altho this predates this PR)